android/app/src/main/java/org/cryptimeleon/incentive/app/domain/model/UserUpdateChoice.kt

```diff
@@ -0,0 +1,58 @@
+package org.cryptimeleon.incentive.app.domain.model
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.modules.SerializersModule
+import kotlinx.serialization.modules.polymorphic
+import kotlinx.serialization.modules.subclass
+import java.math.BigInteger
+import java.util.*
+
+sealed interface UserUpdateChoice
```

Comment:
short comment what these classes/interfaces/objects/... are good for might be good

Reply:
Roger that, added some comments to clarify this